cc/crypto/hpke/recipient_context.cc

```diff
@@ -118,14 +118,14 @@ std::vector<uint8_t> RecipientContext::GenerateNonce() {
   return nonce;
 }
 
-absl::StatusOr<std::string> RecipientContext::Open(absl::string_view ciphertext,
+absl::StatusOr<std::string> RecipientContext::Open(const std::vector<uint8_t>& nonce,
```

Comment:
Related to the previous point: why is this a vector reference rather than a stringview?